stampit.js

```diff
@@ -8,25 +8,30 @@
  **/
 'use strict';
 var forEach = require('mout/array/forEach');
-var mixIn = require('mout/object/mixIn');
-var merge = require('mout/object/merge');
 var map = require('mout/array/map');
 var forOwn = require('mout/object/forOwn');
-var mixInChain = require('./mixinchain.js');
+var deepClone = require('mout/lang/deepClone');
 var slice = [].slice;
 
-// Avoiding JSHist W003 violations.
-var create, extractFunctions, stampit, compose, isStamp, convertConstructor;
+function isFunction(val) {
+  return typeof val === 'function';
+}
 
-create = function (o) {
-  if (arguments.length > 1) {
-    throw new Error('Object.create implementation only accepts the first parameter.');
-  }
-  function F() {}
+var mixer = require('./mixer');
+var mixIn = mixer.getMixin(); // Regular mixin function.
+var mixInFunctions = mixer.getMixin(isFunction); // mixin for functions only.
```

Comment:
This is overly complicated. I'd prefer it if you just export all the various types of functions you need. Keep the DOT (Do One Thing) principle in mind when you write modules. You DOT and DRY don't have to conflict. Each of the separate modules can share dependencies for code reuse. The point is that we shouldn't expose the complexity of configuring mixer functions to the main stampit module. It should be able to simply require the functions it needs by name.

For more on how to keep module interfaces simple, watch the talk [Simplicity Matters by Rich Hickey](https://www.youtube.com/watch?v=rI8tNMsozo0).